machines/VCItemMachine/ExistingMosipVCItem/ExistingMosipVCItemMachine.ts

```diff
@@ -62,7 +62,6 @@ const model = createModel(
     transactionId: '',
     bindingTransactionId: '',
     revoked: false,
-    downloadCounter: 0,
```

Comment:
Since, downloadCounter is not common for all VC's and also it is not useful post download (success or failure), so can we mention it as context only and not as field in vcMetadata.

Moreover vcMetadata can be light weight with only the meta info that is common for all VC's should be avl there.

Reply:
downloadCounter moved to context from vcMetadata